sklearn/linear_model/logistic.py

```diff
@@ -1814,7 +1814,10 @@ class LogisticRegressionCV(LogisticRegression, BaseEstimator,
         If int, random_state is the seed used by the random number generator;
         If RandomState instance, random_state is the random number generator;
         If None, the random number generator is the RandomState instance used
-        by `np.random`.
+        by `np.random`. Note that this only applies to the solver and not the
+        cross-validation generator. Futhermore, if a deterministic solver is
```

Comment:
Remove the sentence after starting by `Furthermore`. It is better just to mentioned the solver for which the `random_state` is used.